packages/admin/src/PrismaTable/Form/Inputs.tsx

```diff
@@ -227,6 +227,13 @@ const defaultInputs: Omit<FormInputs, 'Upload'> = {
       });
     }
 
+    React.useEffect(() => {
+      if (data && data[`findOne${field.type}`]) {
+        setState(data[`findOne${field.type}`]);
```

Comment:
and we don't need to do this pull because it's do nothing we already have this data inside our state.
we make the query to get the relations data for dynamic table